### PR TITLE
[4.0][com_checkin] convert to prepared statement

### PR DESCRIPTION
**Pull request by alikon**
### Summary of Changes
use prepared statement for SQL

### Testing Instructions
test com_checkin

### Expected result

should work as before

### Actual result

N/A

### CODE REVIEW

File: administrator/components/com_checkin/Model/CheckinModel.php [hunk 1]
```diff
@@ -111,8 +111,9 @@ public function checkin($ids = array())
 			$query = $db->getQuery(true)
 				->update($db->quoteName($tn))
 				->set($db->quoteName('checked_out') . ' = DEFAULT')
-				->set($db->quoteName('checked_out_time') . ' = ' . $db->quote($nullDate))
-				->where($db->quoteName('checked_out') . ' > 0');
+				->set($db->quoteName('checked_out_time') . ' = :checkoutime')
```
Comment (SharkyKZ) [inline administrator/components/com_checkin/Model/CheckinModel.php]:
```suggestion
				->set($db->quoteName('checked_out_time') . ' = :checkouttime')
```

**Comment by SharkyKZ** [comment]
I have tested this item :white_check_mark: successfully on d8c66170af43aa4452a2bf4b87a6e780733495cc<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25558">issues.joomla.org/tracker/joomla-cms/25558</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on d8c66170af43aa4452a2bf4b87a6e780733495cc<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25558">issues.joomla.org/tracker/joomla-cms/25558</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25558">issues.joomla.org/tracker/joomla-cms/25558</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**